classes/04class/exercises/c04-iac01/gpavelar/iam.tf

```diff
@@ -0,0 +1,61 @@
+# Create Role - According class C01-AWS02
+resource "aws_iam_role" "terraform_s3_access_role" {
+  name               = "terraform-s3-role"
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
```

Comment:
template_file can help if the policy is big or you need to write big text files.
https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file